### ISSUE TITLE
Feature parity for STEP simulations

### ISSUE THREAD

**Issue by theo-brown** [google-deepmind/torax#420]
In order to run STEP simulations in TORAX we need the following feature parity with RAPTOR and JETTO:

- [x] Arbitrary prescribed profiles
- [x] Bohm-gyro-Bohm transport (#266 #406)
- [ ] Electron-cyclotron heating and current drive (#399)
- [ ] V-loop boundary condition (time derivative of psi_edge = 0)
- [ ] Confinement scaling of `chi` profiles
  - [ ] Bonus: pedestal imposed in `chi` rather than `Te`

**Comment by theo-brown** [comment]
Added three new features from conversation with @fcasson.

**Comment by theo-brown** [comment]
With #456 brought in, we're only really missing neoclassical transport now! The reason for particularly needing it is that in order to do scenario optimisation we need self-consistency, which means that prescribing an inner patch isn't really sufficient.

**Comment by jcitrin** [comment]
Looks like this is ready to go! :)

**Comment by jcitrin** [comment]
Closing since other 2 nice-to-haves are not planned in the immediate-term (but will eventually be done), and the STEP validation exercise can still proceed. Feel free to reopen if needed.

**Comment by theo-brown** [comment]
Yep, super! Been gradually updating the configs as more features have been added, should have time to give a proper go of it in the latter half of July!
